kiali-server/values.yaml

```diff
@@ -113,6 +113,7 @@ login_token:
 
 server:
   port: 20001
+  node_port: 32475
```

Comment:
Remove this, since we want the default to be undefined.
```suggestion
```

Reply:
Do we still want to see the variable in values ? If it is okay for you, we can proceed in this way, of course

Reply:
Actually, that's a good point. As the comments at the top of this values.yml file says:
```
# Settings that mimic the Kiali CR which are placed in the ConfigMap.
# Note that only those values used by the Helm Chart will be here.
```

So values that the helm chart actually reference or use are documented here, even if the default value is undefined. So, yeah, can you just comment that out without a value. We do this in several other places in here, just mimic what is done like these:
* https://github.com/kiali/helm-charts/blob/master/kiali-server/values.yaml#L63
* https://github.com/kiali/helm-charts/blob/master/kiali-server/values.yaml#L98-L99